carta/image.py

```diff
@@ -630,8 +634,8 @@ def use_channel_histogram(self, contours=False):
         self.call_action(f"renderConfig.setUseCubeHistogram{'Contours' if contours else ''}", False)
 
     @validate(Number(0, 100))
-    def set_percentile_rank(self, rank):
-        """Set the percentile rank.
+    def set_clip_percentile(self, rank):
```

Comment:
There is an existing issue about setting the rank not in the preset ranks (90, 95, 99, ...). If we set the rank as 90, the 90% button will be selected (down). But if we set a rank such as 80, no button will be selected. I suggest we select the "Custom" button in such cases. This means we trigger the "Custom" button _after_ we set the clip percentile.

Reply:
@kswang1029 @confluence 
Done in 8f98ffd. Check the result below:
https://drive.google.com/file/d/1N9b4mAfRp5Zqa6aJmX5LDoiNmxK2eKzv/view?usp=sharing

Reply:
@kswang1029, @confluence 
Made a minor modification according to the suggestions of @YuHsuan-Hwang. `setPercentileRank` is called again to just "press the `Custom` button" instead of resetting the colormap with `setCustomScale`.